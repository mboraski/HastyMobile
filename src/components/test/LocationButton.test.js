// import 'react-native';
// import React from 'react';
// import { shallow } from 'enzyme';
//
// import LocationButton from '../LocationButton';
//
// describe('LocationButton', () => {
//     it('renders correctly', () => {
//         const render = shallow(<LocationButton />);
//         expect(render).toMatchSnapshot();
//     });
// });
