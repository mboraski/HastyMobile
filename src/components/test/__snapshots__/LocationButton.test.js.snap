// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LocationButton renders correctly 1`] = `
<Icon
  color="black"
  containerStyle={
    Array [
      Object {
        "backgroundColor": "#F4F4F4",
        "marginRight": 12,
        "shadowColor": "#000",
        "shadowOffset": Object {
          "height": 2,
          "width": 0,
        },
        "shadowOpacity": 0.3,
        "shadowRadius": 1,
      },
      undefined,
    ]
  }
  iconStyle={
    Object {
      "color": "#4c4c4c",
      "transform": Array [
        Object {
          "rotate": "45deg",
        },
      ],
    }
  }
  name="navigation"
  onPress={[Function]}
  raised={true}
  reverse={false}
  reverseColor="white"
  size={25}
  underlayColor="white"
/>
`;
